### PR TITLE
Sams branch

### PR DESCRIPTION
**Pull request by samjiangweb**
Tweaks to colours, fonts, and other interface elements. Also fixed the teaching page layout.

### CODE REVIEW

File: src/Layout.scss [hunk 1]
```diff
@@ -36,11 +48,11 @@
 }
 
 .nav-bar-component {
-  background-color: #3f51b2;
+  background-color: $black;
   height: 64px;
-  -webkit-box-shadow: 0px 2px rgba(0, 0, 0, 0.25);
+  /*-webkit-box-shadow: 0px 2px rgba(0, 0, 0, 0.25);
```
Comment (davidascher) [inline src/Layout.scss]: Let's avoid leaving commented out stuff.

File: src/About.scss [hunk 2]
```diff
@@ -77,7 +81,7 @@
       }
 
       .name {
-        color: $color2;
+        color: $orange;
```
Comment (andreadelrio) [inline src/About.scss]: @samjianga consider using generic variable names such as primary-color, secondary-color, etc. That way if we change the colors later that variable name still makes sense.

File: src/Layout.scss (hunk 1 above)
Comment (andreadelrio) [inline src/Layout.scss]: Remove vendor specific prefixes, they are supposed to be added automatically when we compile the scss.

File: src/Layout.scss [hunk 3]
```diff
@@ -53,12 +65,18 @@
   .logo-area {
     float: left;
     width: 320px;
+    @media screen and (min-width:1100px){
```
Comment (andreadelrio) [inline src/Layout.scss]: For media queries we use Bootstrap4's built-in media queries. See: 
https://v4-alpha.getbootstrap.com/layout/overview/#responsive-breakpoints and use mixins like @include media-breakpoint-down(xs) { ... } as needed.

File: src/components/SearchResultsView/SearchResultsView.scss [hunk 4]
```diff
@@ -50,24 +51,36 @@
 }
 
   .search-actions-area {
-    border-bottom: 2px solid #eee;
-    padding-top: 32px;
+    //border-bottom: 2px solid #f3efec;
```
Comment (andreadelrio) [inline src/components/SearchResultsView/SearchResultsView.scss]: remove line

File: src/About.scss (hunk 2 above)
Comment (andreadelrio) [inline src/About.scss]: If you want to keep names that describe the colours you can do something like:
```
$red: some hex code;
$primary-color: $red;
```

File: src/components/SearchResultsView/SearchResultsView.scss [hunk 5]
```diff
@@ -50,24 +51,36 @@
...
+      font-weight: 600;
+      text-transform:uppercase;
+      color: $black;
       display: block;
       line-height: 26px;
       float: left;
-      padding: 0 4px 8px 4px;
-      margin-right: 24px;
+      padding: 14px 20px 10px;
+      //margin-right: 24px;
```
Comment (andreadelrio) [inline src/components/SearchResultsView/SearchResultsView.scss]: remove

File: src/global.scss [hunk 6]
```diff
@@ -1,3 +1,22 @@
+@font-face {
+  font-family:'Fakt';
+    src: url('fonts/FaktPro-Medium.ttf');
```
Comment (andreadelrio) [inline src/global.scss]: check indentation

**Pull request closed without merging**